<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilenamePanel</class>
 <widget class="QWidget" name="FilenamePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>94</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>16</number>
   </property>
   <property name="topMargin">
    <number>16</number>
   </property>
   <property name="rightMargin">
    <number>16</number>
   </property>
   <property name="bottomMargin">
    <number>16</number>
   </property>
   <item row="0" column="1" alignment="Qt::AlignTop">
    <widget class="QPushButton" name="browsePushButton">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" alignment="Qt::AlignTop">
    <widget class="nx::vms::client::desktop::AdvancedComboBox" name="extensionsComboBox">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="nx::vms::client::desktop::InputField" name="folderInputField" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="nx::vms::client::desktop::InputField" name="nameInputField" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>nx::vms::client::desktop::AdvancedComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/advanced_combo_box.h</header>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::InputField</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/input_field.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
